src/screen/tips/TipDetailScreen.tsx

```diff
@@ -128,7 +128,7 @@ function TipDetailScreen({route}: ScreenProps) {
         data={tip.tips}
         style={[globalStyles.paddedContainer, styles.container]}
         ItemSeparatorComponent={Divider}
-        renderItem={({item}) => {
+        renderItem={({item}: {item: any[]}) => {
```

Comment:
Why are we having this in every PR? 😆

Reply:
it doesn't let me commit if I don't fix this :D

Reply:
done ✅